indra/llimage/llimagedxt.cpp

```diff
@@ -443,23 +438,23 @@ bool LLImageDXT::convertToDXR()
 
 	LLImageDataLock lock(this);
 
-	S32 width = getWidth(), height = getHeight();
-	S32 nmips = calcNumMips(width,height);
-	S32 total_bytes = getDataSize();
 	U8* olddata = getData();
+	llassert_return_false(olddata);
+	S32 total_bytes = getDataSize();
+	llassert_return_false(total_bytes > 0);
 	U8* newdata = (U8*)ll_aligned_malloc_16(total_bytes);
-	if (!newdata)
-	{
-		LL_ERRS() << "Out of memory in LLImageDXT::convertToDXR()" << LL_ENDL;
-		return false;
-	}
-	llassert(total_bytes > 0);
+	llassert_message_return_false(newdata, "Out of memory in LLImageDXT::convertToDXR()")
```

Comment:
@akleshchev / @marchcat-pe make good points about how to handle OOM - this is another example here